### PR TITLE
Update user model unit tests

### LINKED ISSUE #536: Write unit tests for the User model
Write unit tests for the [`User` Model](https://github.com/code-corps/code-corps-ember/blob/develop/app/models/user.js). There is an [existing unit test file](https://github.com/code-corps/code-corps-ember/blob/develop/tests/unit/models/user-test.js), but it could use more coverage (computed properties & relationship declarations). 

## References
- Progress on: #98 
- [Model testing guide](https://guides.emberjs.com/v2.8.0/testing/testing-models/)
- [Contributing Guidelines](https://github.com/code-corps/code-corps-ember/blob/develop/.github/CONTRIBUTING.md)
- Join our [Slack](https://codecorps.slack.com)

### PR DESCRIPTION
**Pull request by kumarmj**
fixes #536

### CODE REVIEW

File: tests/unit/models/user-test.js [hunk 1]
```diff
@@ -16,3 +17,47 @@ test('it exists', function(assert) {
...
+  assert.ok(attributes.includes('lastName'), 'should have lastName attribute');
+  assert.ok(attributes.includes('name'), 'should have name attribute');
+  assert.ok(attributes.includes('password'), 'should have password attribute');
+  assert.ok(attributes.includes('photoLargeUrl'), 'should have photoLargeUrl attribute');
+  assert.ok(attributes.includes('photoThumbUrl'), 'should have photoThumbUrl attribute');
+  assert.ok(attributes.includes('state'), 'should have state attribute');
+  assert.ok(attributes.includes('twitter'), 'should have twitter attribute');
+  assert.ok(attributes.includes('username'), 'should have username attribute');
+  assert.ok(attributes.includes('website'), 'should have website attribute');
+  assert.ok(attributes.includes('stateTransition'), 'should have stateTransition attribute');
```
Comment (JoshSmith) [inline tests/unit/models/user-test.js]: Would put this right after `state`.

File: tests/unit/models/user-test.js [hunk 2]
```diff
@@ -16,3 +17,47 @@ test('it exists', function(assert) {
...
+
+test('it has all of its attributes', function(assert) {
+  assert.expect(15);
+
+  const user = this.subject();
+  const attributes = Object.keys(user.toJSON());
+
+  assert.ok(attributes.includes('base64PhotoData'), 'should have base64PhotoData attribute');
+  assert.ok(attributes.includes('biography'), 'should have biography attribute');
+  assert.ok(attributes.includes('insertedAt'), 'should have insertedAt attribute');
```
Comment (WenInCode) [inline tests/unit/models/user-test.js]: Can you move this one below the `firstName` attribute check (alpha ordering)

**Comment by WenInCode** [comment]
LGTM @kumarmj - Thanks a bunch! Do you have merge access? If so you are good to merge. (/cc @JoshSmith)

**Comment by JoshSmith** [comment]
Thanks @kumarmj! 🙌

**Comment by kumarmj** [comment]
got great feeling of enjoyment, now looking for more issues

**Pull request merged**